api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -537,6 +537,10 @@ def serialize_collective_offer(cls, collective_offer: educational_models.Collect
                 "name": venue.name,
                 "publicName": venue.publicName,
             },
+            "_geoloc": {
+                "lat": venue.latitude,
+                "lng": venue.longitude,
+            },
```

Comment:
Pour les offres individuelles, on utilise la fonction `position()` (définie dans ce module) qui a des valeurs par défaut quand le lieu n'a pas de coordonnées géographiques. On voudrait sûrement la même chose pour les offres collectives (et les offres vitrines), qu'en penses-tu ?

```suggestion
            "_geoloc": position(venue),
```

Idem plus bas dans `serialize_collective_offer_template()`.

Reply:
Si une personne recherche ensuite les offres collectives dans un périmètre qui inclut le point par défaut, elle va donc voir toutes ces offres ? Ce qui risque d'être assez déroutant non ?

Reply:
Il y a quelque chose que je ne comprends pas : pourquoi une valeur par défaut plutôt que `None` ? Cela change quelque chose au niveau de l'index ? Des calculs ?

Reply:
D'après l'historique, l'ajout est lié au ticket PC-3862, qui a des détails.

Reply:
Après discussion en interne, on ne veut justement pas de valeur par défaut. Donc je propose de laisser tel quel. Et au pire on corrigera après.